File: 2129-capitalize-the-title/2129-capitalize-the-title.js
/**
 * @param {string} title
 * @return {string}
 */
var capitalizeTitle = function(title) {
    
    const str = title.toLowerCase();
    const arr = str.split(' ');
    arr.forEach((ele, i, arr)=>{
        if(ele.length>2)
        arr[i]=ele[0].toUpperCase()+ele.substring(1, ele.length);
    })
    
    return arr.join(' ');
};

File: 2129-capitalize-the-title/2129-capitalize-the-title.cpp
class Solution {
public:
    string capitalizeTitle(string title) {
        
        int n=title.size();
        int j=0;
        for(int i=0;i<n;i++){
            
            
            if(title[i]==' '){
                int len = i-j;
                
                if(len>=3){
                    title[j] = toupper(title[j]);
                    
                }
                j=i+1;
            }else{
                title[i] = tolower(title[i]);
            }
        }
        if(title.size()-j>=3) title[j]=toupper(title[j]);
        return title;
    }
};